src/main/webapp/WEB-INF/themes/desktop/resource/js/pages/aperta_article.js

```diff
@@ -39,6 +39,12 @@ $(document).ready(function () {
     $("#authors").addClass("section-box");
     $("#authors").insertAfter(h2);
 
+    $(".author-corresponding").each(function(index, item) {
+        var html = $(item).html();
```

Comment:
I think javascript is not the right place for display details such as these. Let's put them in the css layer. it's also a less expensive operation.

Reply:
done.